master/docs/relnotes/0.8.10.rst

```diff
@@ -0,0 +1,82 @@
+Release Notes for Buildbot 0.8.10
+=================================
+
+..
+    Any change that adds a feature or fixes a bug should have an entry here.
+    Most simply need an additional bulleted list item, but more significant
+    changes can be given a subsection of their own.
+
+The following are the release notes for Buildbot 0.8.10.
+Buildbot 0.8.10 was released on the 2nd of December, 2014.
+
+Master
+------
+
+Features
+~~~~~~~~
+
+* Both the P4 source step and P4 change source support ticket-based authentication.
+
+* Clickable 'categories' links added in 'Waterfall' page (web UI).
+
+Fixes
+~~~~~
+
+* Buildbot is now compatible with SQLAlchemy 0.8 and higher, using the newly-released SQLAlchemy-Migrate.
```

Comment:
Thanks. By the way, I've tested the new version of alembic, when used with the latest sqlalchemy it does marvelous things. I still want to give a try to port buildbot to alembic ;)